src/price/ProgressivePriceThresholdsDto.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace hiqdev\php\billing\price;
+
+use Money\Currency;
+
+class ProgressivePriceThresholdsDto
+{
+   public float $price;
```

Comment:
Why public?

Reply:
Why not? We have a public properties in all DTO class

Reply:
Makes sense. I wrote this question, treating the class as if it is already a VO. Sorry